WaterMinder/History/Cell/Model/LiquidType.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  LiquidType.swift
+//  WaterMinder
+//
+//  Created by Yuliya on 15/12/2022.
+//
+
+import Foundation
+import UIKit
+
+enum LiquidType {
+    
+    case water
+    case coffee
+    case tea
+    
+    var typeModel: HistoryCollectionViewModel {
```

Comment:
it's not correct, you should only return image and title from enum, not model which is connected to the specific screen